src/pager.ts

```diff
@@ -8,6 +8,7 @@ export type PageLink =
 // Pager links in the footer are computed once for a given navigation.
 const linkCache = new WeakMap<Config["pages"], Map<string, PageLink>>();
 
+/** Strip URL query string and hash fragment. */
```

Comment:
```suggestion
// Strip URL query string and hash fragment.
```

Reply:
These two types of comments are not equivalent. The `/**` style that I wrote causes the documentation to be available the call site, which is where I needed. Using a `//` style comment only makes this documentation visible at the implementation site, and if I'm already looking at the implementation, it is very clear what this function is doing.